layouts/header.css

```diff
@@ -10,6 +10,7 @@
 }
 
 input:checked + .overlay,
+input:checked + label + .overlay,
```

Comment:
This is great like this! We should have the same markup structure everywhere (i.e. `input` before `label`), but since we don't this solves this styling issue very nicely!